Formula/mruby-cli.rb

```diff
@@ -0,0 +1,35 @@
+class MrubyCli < Formula
+  desc "Build native command-line applications for Linux, MacOS, and Windows"
+  homepage "https://github.com/hone/mruby-cli"
+  url "https://github.com/hone/mruby-cli/archive/v0.0.4.tar.gz"
+  sha256 "97d889b5980193c562e82b42089b937e675b73950fa0d0c4e46fbe71d16d719f"
+
+  def install
+    ENV["MRUBY_CLI_LOCAL"] = "true"
+    rewrite_build_config
+    system "rake", "compile"
+    cp "mruby/build/host/bin/mruby-cli", "mruby-cli"
+    bin.install "mruby-cli"
+  end
+
+  # The upstream build_config.rb assumes it will be compiled on Linux.
+  # It also automatically cross compiles to MacOS, Linux, and Windows
+  # This configuration compiles a single MacOS binary using clang
+  def rewrite_build_config
```

Comment:
Instead of making this a method, just inline it into `#install`:
```ruby
rm buildpath/"build_config.rb"

(buildpath/"build_config.rb").write <<-EOS.undent
  MRuby::Build.new do |conf|
    toolchain :clang
    conf.gem File.expand_path(File.dirname(__FILE__))
  end
EOS